### PR TITLE
Adds silent option to ignore cmake output

### PR DESCRIPTION
**Pull request by gerhardberger**
This PR adds a `silent` option to ignore the cMake's output on the stdio.

### CODE REVIEW

File: bin/cmake-js [hunk 1]
```diff
@@ -112,6 +112,11 @@ var yargs = require("yargs")
             demand: false,
             describe: "Custom argument passed to CMake in format: -D<your-arg-here>",
             type: "string"
+        },
+        "si": {
```
Comment (unbornchikken) [inline bin/cmake-js]: An option key should be one-letter in yargs.

File: lib/es6/processHelpers.js [hunk 2]
```diff
@@ -5,12 +5,12 @@ let spawn = require("child_process").spawn;
 let exec = require("child_process").exec;
 
 let processHelpers = {
-    run: function (command) {
+    run: function (command, silent) {
```
Comment (unbornchikken) [inline lib/es6/processHelpers.js]: There should be no explicit options arguments on utility helpers. Please use an options object, which have a silent property with default value of "inherit" (by using lodash's _.defaults).

File: README.md [hunk 3]
```diff
@@ -143,6 +143,7 @@ Options:
                                                                        [boolean]
   --CD                   Custom argument passed to CMake in format:
                          -D<your-arg-here>                              [string]
+  --si, --silent         Prevents cMake to print to the stdio          [boolean]
```
Comment (unbornchikken) [inline README.md]: Please change "Prevents cMake" to "Prevents CMake.js".

**Comment by gerhardberger** [comment]
fixed them, thanks!

**Pull request merged**